File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/domain/model/ParkingZoneGeometry.java
package com.lattis.ellipse.domain.model;

/**
 * Created by lattis on 24/05/17.
 */

public class ParkingZoneGeometry {
    private double latitude;
    private double longitude;
    private double radius;

    public double getRadius() {
        return radius;
    }

    public void setRadius(double radius) {
        this.radius = radius;
    }



    public double getLatitude() {
        return latitude;
    }

    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }

    public double getLongitude() {
        return longitude;
    }

    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }

    @Override
    public String toString() {




        return "ParkingZoneGeometryResponse{" +
                "latitude=" + latitude +
                "longitude=" + longitude +
                '}';

    }
}

File: velolabs_repos/lattis_android_v2/domain/src/main/java/com/lattis/domain/models/ParkingZoneGeometry.kt
package com.lattis.domain.models

class ParkingZoneGeometry {
    var latitude: Double = 0.toDouble()
    var longitude: Double = 0.toDouble()
    var radius: Double = 0.toDouble()

    override fun toString(): String {


        return "ParkingZoneGeometryResponse{" +
                "latitude=" + latitude +
                "longitude=" + longitude +
                '}'.toString()

    }
}
